package.json

```diff
@@ -171,9 +172,11 @@
     "redux-devtools-extension": "^2.13.2",
     "redux-persist": "^6.0.0",
     "redux-thunk": "^2.2.0",
+    "reflect-metadata": "^0.1.13",
     "reselect": "^4.0.0",
     "rn-fetch-blob": "^0.12.0",
     "rn-nodeify": "^10.0.1",
+    "rxjs": "^6.6.2",
```

Comment:
Is this dep actually used?

Reply:
It's required on Etherspot SDK install https://github.com/etherspot/etherspot-sdk#installation, however, it was used in full implementation branch, but just tried to remove it here and it works just fine without – included changes, thanks 👍  P. S. same goes for `reflect-metadata`, however removing it causes missing dependency issues